src/main/java/com/violeth/blockjs/blockjs/commands/VersionCommand.java

```diff
@@ -0,0 +1,22 @@
+package com.violeth.blockjs.blockjs.commands;
+
+import com.violeth.blockjs.blockjs.BlockJS;
+import com.violeth.blockjs.blockjs.jsinterface.JSRunner;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.TextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+import org.bukkit.command.CommandSender;
+
+public class VersionCommand extends Command {
+    {
+        key = "version";
+    }
+    @Override
+    public void handle(CommandSender sender, String[] args) {
+        sender.sendMessage(BlockJS.getPluginMessagePrefix().append(
+            Component.text("The current version: ").append(
+                Component.text(BlockJS.instance.getDescription().getVersion()).color(TextColor.color(0xDDDDDD)).decorate(TextDecoration.ITALIC)
```

Comment:
There also isn't an `instance` field on the BlockJS class

Reply:
I'm assuming what you were doing was trying to get the version, which in that case you can use something like `Bukkit.getServer().getPluginManager().getPlugin("BlockJS").getDescription().getVersion()` to get the version

Reply:
I have no clue what happened to the commited file, but here it is